python/cuml/multiclass/multiclass.py

```diff
@@ -115,6 +120,17 @@ def classes_(self):
     def n_classes_(self):
         return self.multiclass_estimator.n_classes_
 
+    def _input_to_host_array(self, X):
+        _array_type, is_sparse = determine_array_type_full(X)
+        if is_sparse:
+            if _array_type == "cupy":
+                X = SparseCumlArray(X).to_output(output_type="scipy")
+            elif _array_type == "cuml":
+                X = X.to_output(output_type="scipy")
+        else:
+            X = input_to_host_array(X).array
+        return X
```

Comment:
I would be more defensive here and instead of the `X = ...` pattern, I would just return directly.
```suggestion
    def _input_to_host_array(self, X):
        _array_type, is_sparse = determine_array_type_full(X)
        if is_sparse:
            if _array_type == "cupy":
                return SparseCumlArray(X).to_output(output_type="scipy")
            elif _array_type == "cuml":
                return X.to_output(output_type="scipy")
            else:
                raise ValueError(f"{Unsupported sparse array type: {_array_type}.")
        return input_to_host_array(X).array
```